Split the comment

garretthyder opened the pull request:
## Description
Separate the comment so translator comment is in it's own block.

## How has this been tested?
This is a change to comments so aside from loading to make sure nothing broke the test will be done as part of parsing.

## Types of changes
Bug fix #12139 corrects the translator comments for i18n

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. <!-- Check code: `npm run lint`, Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/ -->
- [x] My code follows the accessibility standards. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/accessibility-coding-standards/ -->
- [ ] My code has proper inline documentation. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/ -->

Commit:
Split the comment

Seperate the comment so translator comment is in it's own block.

swissspidy commented on lib/client-assets.php:
```diff
@@ -1211,8 +1211,8 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
+	// Set a locale specific default font.
 	/*
-	 * Set a locale specific default font.
 	 * Translators: Use this to specify the CSS font family for the default font
```
PHPCS currently fails: `Empty line required before block comment`.

The `Set a locale specific default font.` comment is a bit redundant as the translator comment kinda explains that anyway.

I'd simply remove it.

Also, I'd write the translator comment like this:

```php
/* Translators: Use this to specify the CSS font family for the default font. */
```

garrett-eclipse replied:
Thanks @swissspidy make complete sense thanks, I've adjusted accordingly.

Commit:
Updated comments to address review

Updated the comments to address @swisspidy's review.
Removed the redundant comment and switched to a single line comment block.
